integration-tests/index.ts

```diff
@@ -16,8 +16,9 @@ import './certificate';
 
 import './auth';
 import './settings';
-import './course';
 import './appointments';
```

Comment:
The appointments test case depends on course, thus it deadlocks when appointments is run before course.

Reply:
fixed it :)